scripts/android-build-common.sh

```diff
@@ -12,7 +12,7 @@ if [ -z $NDK_TARGET ]; then
 fi
 
 if [ -z $CMAKE_PROGRAM ]; then
-  	CMAKE_PROGRAM=$(find $ANDROID_SDK/cmake -name cmake -type f -executable -print -quit)
+  	CMAKE_PROGRAM=$(find $ANDROID_SDK/cmake -name cmake -type f -print -quit)
```

Comment:
sorry, this breaks the sdk/ndk stuff on our build machines :/

Reply:
```
Command /opt/android-sdk-linux/cmake/3.10.2.4988404/share/cmake-3.10/completions/cmake not found. Install and add it to the PATH
```
@akallabeth did my change caused it?

Reply:
yes. it finds the first file matching `cmake` but that is not necessarily an `executable`

Reply:
reverted. can you review it again?